I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 12, in script
    define y = Character("You").color('red')
  File "game/script.rpy", line 12, in script
    define y = Character("You").color('red')
  File "game/script.rpy", line 12, in <module>
    define y = Character("You").color('red')
AttributeError: 'ADVCharacter' object has no attribute 'color'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/Users/aiko/Desktop/renpy-8.0.3-sdk/renpy/bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "/Users/aiko/Desktop/renpy-8.0.3-sdk/renpy/main.py", line 558, in main
    renpy.game.context().run(node)
  File "game/script.rpy", line 12, in script
    define y = Character("You").color('red')
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "game/script.rpy", line 12, in script
    define y = Character("You").color('red')
  File "/Users/aiko/Desktop/renpy-8.0.3-sdk/renpy/ast.py", line 2441, in execute
    self.set()
  File "/Users/aiko/Desktop/renpy-8.0.3-sdk/renpy/ast.py", line 2455, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "/Users/aiko/Desktop/renpy-8.0.3-sdk/renpy/python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 12, in <module>
    define y = Character("You").color('red')
AttributeError: 'ADVCharacter' object has no attribute 'color'

macOS-10.15.2-x86_64-i386-64bit x86_64
Ren'Py 8.0.3.22090809
Nature Calls 1.0
Sat Mar  4 17:09:26 2023
